linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/tests/correct.pass.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
-chown 0:0 /boot/grub2/grub.cfg
+if [ -e /boot/grub2 ]; then
```

Comment:
Does anybody know whether we can use Jinja macros also in the test scenarios?

Reply:
We can't (yet). That is something I'm working on enabling, to eventually also support templated test cases. :-) I think it requires some changes to the API. See:

https://github.com/ComplianceAsCode/content/blob/master/tests/ssg_test_suite/common.py#L236-L253

Reply:
I plan to revisit this shortly now that #7210 has merged. :-)

Reply:
@jan-cerny This has been updated.